homeworks/HW1/src/InputValidatedField.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {Input} from 'reactstrap';
+import './App.css';
+
+export class InputValidatedField extends React.Component {
+
+    constructor(props) {
+        super(props);
+        const error = {
+            message: null
```

Comment:
Why object if you using the only message?
I guess more preferable string type with the name of var like an `errorMessage`.